core/src/main/java/org/infinispan/xsite/statetransfer/XSiteStateConsumerImpl.java

```diff
@@ -54,7 +52,7 @@
 
    @Override
    public void startStateTransfer(String sendingSite) {
-      if (debug) {
+      if (log.isDebugEnabled()) {
```

Comment:
I don't think `if (log.isDebugEnabled())` is justified, there are just too few invocations.